convert-it/ci.yml

```diff
@@ -0,0 +1,12 @@
+prepare:
+  steps:
+    - name: Installing the python dependencies.
+      command: pipenv install -r requirements.txt
+test:
+  steps: []
+run:
+  steps:
+  - name: Switch to the new python environment.
+    command: source "$(pipenv --venv)/bin/activate"
+  - name: Start the FastAPI server.
+    command: uvicorn main:app --reload --host 0.0.0.0 --port 3000
```

Comment:
this CI-Pipeline doesnt work. Instead having two commands you need to do both in one command.

I changed it already on Codesphere in your workspace. You need to push it to the github repo though.

Reply:
Thank you for pointing that out. I noticed that today as well and was going to ask about that. Will make the changes in the repo then!